src/js/timeline.js

```diff
@@ -0,0 +1,20 @@
+function getTimeline(){
+	
+	var xhttp = new XMLHttpRequest();
+	var URL = "http://localhost:8080/api/1.0/twitter/timeline";
+    xhttp.onreadystatechange = function() {
+         if (this.readyState == XMLHttpRequest.DONE && this.status == 200) {
+            document.getElementById('timelinePlaceholder').innerHTML = this.responseText; 
+         }
+         else if(this.readyState == 3){
```

Comment:
you're likely never going to see the' Please Wait, retrieving timeline.' as the code is barely executed.  If you want it to say that, better to explicitly do it before the ajax call is made.  You could just remove it as well and that would be fine